eslint/babel-eslint-parser/src/client.cjs

```diff
@@ -73,10 +73,17 @@ exports.WorkerClient = class WorkerClient extends Client {
       );
 
       Atomics.wait(this.#signal, 0, 0);
-      const { message } = WorkerClient.#worker_threads.receiveMessageOnPort(
-        subChannel.port2,
-      );
 
+      let resp;
+      for (let i = 0; i < 100; i++) {
+        resp = WorkerClient.#worker_threads.receiveMessageOnPort(
+          subChannel.port2,
+        );
+        if (resp) break;
+        Atomics.wait(this.#signal, 1, 0, 30);
```

Comment:
```suggestion
        Atomics.wait(this.#signal, 1, 0, 5);
```

I guess 30ms is too much? Maybe 5ms is better.

It should be enough to wait only 5 milliseconds once in most cases.

Waiting for a total of 500ms should be equally fine.